test/suite/test_tiered19.py

```diff
@@ -151,6 +151,9 @@ def test_ss_azure_file_system(self):
         ss.ss_customize_file_system(
             session, self.bucket, None, self.get_fs_config(prefix_2))
 
+        # Check fs exist for non-existing object.
+        self.assertFalse(azure_fs_1.fs_exist(session, 'foobar'))
```

Comment:
Do we only have one azure filesystem open? If so i would just rename it to `azure_fs`

Reply:
done :)